src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -420,8 +420,10 @@ private static bool GetProtocolSupportFromWindowsRegistry(SslProtocols protocol,
 #pragma warning disable CS0618 // Ssl2 and Ssl3 are obsolete
                 SslProtocols.Ssl3 => "SSL 3.0",
 #pragma warning restore CS0618
+#pragma warning disable SYSLIB0039 // TLS versions 1.0 and 1.1 have known vulnerabilities
```

Comment:
I don't know why this doesn't give an error when built using the 'default' configuration.

This is what I get:

```
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(424,17): error SYSLIB0039: 'SslProtocols.Tls' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(425,17): error SYSLIB0039: 'SslProtocols.Tls11' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(508,62): error SYSLIB0039: 'SslProtocols.Tls' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(511,41): error SYSLIB0039: 'SslProtocols.Tls' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(521,66): error SYSLIB0039: 'SslProtocols.Tls11' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(525,62): error SYSLIB0039: 'SslProtocols.Tls11' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs(533,41): error SYSLIB0039: 'SslProtocols.Tls11' is obsolete: 'TLS versions 1.0 and 1.1 have known vulnerabilities and are not recommended. Use a newer TLS version instead, or use SslProtocols.None to defer to OS defaults.' [/home/tmds/repos/runtime/src/libraries/Common/tests/TestUtilities/TestUtilities.csproj::TargetFramework=net8.0]
```

Reply:
These were made obsolete in https://github.com/dotnet/runtime/issues/65546.
It seems appropriate to add these.

cc @rzikm @wfurt

Reply:
Interesting, Yes, the warning should be disabled for tests.